### ISSUE TITLE
Network potentially has a loop

### ISSUE THREAD

**Issue by mandoonandy** [shakenfist/shakenfist#310]
Does this indicate the networking has a loop or is ping broken?

Note that the RTT increases by the first RTT value for each duplicate. Cool hey.
```
login as 'cirros' user. default password: 'gocubsgo'. use 'sudo' for root.
bobnet login: cirros
Password: 
$ ping 8.8.8.8
PING 8.8.8.8 (8.8.8.8): 56 data bytes
64 bytes from 8.8.8.8: seq=0 ttl=248 time=6.767 ms
64 bytes from 8.8.8.8: seq=0 ttl=248 time=12.547 ms (DUP!)
64 bytes from 8.8.8.8: seq=0 ttl=248 time=18.858 ms (DUP!)
64 bytes from 8.8.8.8: seq=0 ttl=248 time=24.873 ms (DUP!)
64 bytes from 8.8.8.8: seq=1 ttl=248 time=5.753 ms
64 bytes from 8.8.8.8: seq=1 ttl=248 time=11.265 ms (DUP!)
64 bytes from 8.8.8.8: seq=1 ttl=248 time=18.062 ms (DUP!)
64 bytes from 8.8.8.8: seq=1 ttl=248 time=27.206 ms (DUP!)
64 bytes from 8.8.8.8: seq=2 ttl=248 time=6.877 ms
64 bytes from 8.8.8.8: seq=2 ttl=248 time=23.430 ms (DUP!)
64 bytes from 8.8.8.8: seq=2 ttl=248 time=41.534 ms (DUP!)
64 bytes from 8.8.8.8: seq=2 ttl=248 time=62.740 ms (DUP!)
```
This cluster is installed on clean OS installs. No legacy network setup from previous SF versions.

Same DUP when pinging another VM on same net.

**Comment by mikalstill** [comment]
This has been happening for a while and seems to only happen on cirros. I haven't been able to work out what is causing it...

**Comment by mandoonandy** [comment]
Fixed #861
